modules/dbfs/filestorage.go

```diff
@@ -0,0 +1,57 @@
+package dbfs
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/CodeCollaborate/Server/modules/config"
+)
+
+var filePathSeparator = strconv.QuoteRune(os.PathSeparator)[1:2]
```

Comment:
Are we still using filepathseparators? Is everything being changed to filepath.join?